pegasus/sites.v3/code.org/views/learn_carousels.haml

```diff
@@ -34,8 +34,8 @@
 
 -if variation == 'learn'
   %h1
-    %a{:href=>"http://hourofcode.com/resources/how-to"}
-      = I18n.t(:how_to_teach_hoc)
+    %a{:href=>"https://code.org/teacher-led"}
```

Comment:
if this is on code.org we can just use /teacher-led, then each staging etc. will work properly too.